### DISCUSSION TITLE
Rotate Dropdown list automatically and select option randomly from dashboard variable

### DISCUSSION THREAD

**Discussion by adeel86** [grafana/grafana#63088] [Dashboards, Panels & Visualizations]
Grafana v9.3.2

I’ve created a playlist for multiple dashboards and each dashboard also has a dropdown list. I want to show different network device on the same dashboard when the time comes for that dashboard.

Is there any way that I can achieve it? I looked all over the internet, but I could not find it.

Your assistance will be greatly appreciated.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
